pkgs/by-name/pr/project-lemonlime/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsForQt5,
+  qt5,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "project-lemonlime";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "Project-LemonLime";
+    repo = "Project_LemonLime";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-h/aE1+ED+RkXqFcsb23rboA+Dd7kiom3XiIRqb4oYkQ=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    qt5.wrapQtAppsHook
```

Comment:
Qt 6 should be preferred if there are no issues with it. Upstream provided https://github.com/Project-LemonLime/Project_LemonLime/blob/master/CMakeLists.txt#L47

(Also qt6.qtwayland!)